htdocs/ticket/class/ticket.class.php

```diff
@@ -2598,7 +2598,7 @@ public function newMessage($user, &$action, $private = 1, $public_area = 0)
 							$subject = '['.$label_title.'- ticket #'.$object->track_id.'] '.$langs->trans('TicketNewMessage');
 
 							// Message send
-							$message = $langs->trans('TicketMessageMailIntroText');
+							$message = getDolGlobalString('TICKET_MESSAGE_MAIL_INTRO');
```

Comment:
What if TICKET_MESSAGE_MAIL_INTRO is not configured/empty ?
May be we can use
`$message = getDolGlobalString('TICKET_MESSAGE_MAIL_INTRO', $langs->trans('TicketMessageMailIntroText'));`

Reply:
I agree and it's done. If someone really wants the intro message to disappear he can always input a space character or something similar.